Project.toml

```diff
@@ -97,6 +100,7 @@ Triangulate = "2.2"
 TriplotBase = "0.1"
 TriplotRecipes = "0.1"
 TrixiBase = "0.1.1"
+UUIDs = "1.6"
```

Comment:
UUIDs needs to be skipped in the Downgrade.yml since it is a stdlib.

Reply:
Thanks! Done in https://github.com/trixi-framework/Trixi.jl/pull/1781/commits/c5eee3567ff1b458f0dba83840b81aece21121b3